doc/script_commands.txt

```diff
@@ -8950,6 +8950,21 @@ Valid types are:
    0: Elemental ID
    1: Elemental Game ID
 
+
+*setbodysize <GID>, <size value>;
```

Comment:
it may be a matter of preference but the parameters are similar. How about adding a new constant for setunitdata?
Example: `setunitdata $@mobid[0], UMOB_BODYSIZE, rand(10, 300);`

new constant : `UMOB_BODYSIZE`